src/main/java/org/onebusaway/alexa/constant/SessionAttribute.java

```diff
@@ -1,25 +1,24 @@
 /*
  * Copyright 2016 Sean J. Barbeau (sjbarbeau@gmail.com),
  * Philip M. White (philip@mailworks.org)
- *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
  *      http://www.apache.org/licenses/LICENSE-2.0
- *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.onebusaway.alexa;
+package org.onebusaway.alexa.constant;
 
 /**
- * OneBusAway Alexa session attributes
+ * OneBusAway Alexa session attributes.
  */
 public class SessionAttribute {
+    //PRINCIPLE_ID in session can be personId or userId depends which one is available.
```

Comment:
Nit - all `//` should have a following space ` `. So this should be `// PRINCIPLE_ID`

Reply:
Done